bpf-gpl/bpf.h

```diff
@@ -163,6 +163,7 @@ enum calico_skb_mark {
 	CALI_SKB_MARK_BYPASS                 = CALI_SKB_MARK_SEEN | 0x10000,
 	CALI_SKB_MARK_BYPASS_FWD             = CALI_SKB_MARK_SEEN | 0x30000,
 	CALI_SKB_MARK_BYPASS_FWD_SRC_FIXUP   = CALI_SKB_MARK_SEEN | 0x50000,
+	CALI_SKB_MARK_SKIP_RPF               = CALI_SKB_MARK_SEEN | 0x40000,
```

Comment:
The 4-bit overlaps with the 5 above, doe we always check explicitly for the 5, or do we treat the bypass and src fixup bits separately?

Reply:
Except pure CALI_SKB_MARK_SEEN (which is a prefix), we always test the whole value. We do not have more bits unless we claim more :-/